packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -24,16 +26,24 @@ const messages = {
   description: 'Description'
 }
 
-export const TrackMetadataFields = () => {
+type TrackMetadataFieldsProps = {
+  /** Whether or not the preview is playing. */
+  playing: boolean
+  index: number
+}
+
+export const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
+  const { index } = props
```

Comment:
would we want to put index in some context so we dont have to pass a bunch of stuff down? not convinced that is a good idea, but i wonder if establishing a multi-track hierarchy via context some other mech would be compelling?

Reply:
maybe a `useSomethingField` that has context of index and can handle it all for us? that way we dont have to pass index around and call `getTrackFieldName` for every field

Reply:
I thought about context, but field hooks is another interesting idea... I'll think about it